bso/structural_design/element/element.cpp

```diff
@@ -141,6 +141,16 @@ namespace bso { namespace structural_design { namespace element {
 		throw std::runtime_error("cannot call getStressCenter() for this element type");
 	} // getStressCenter() gives error (standard) except for elements in which the function is overridden
 
+	Eigen::VectorXd element::getStressSensitivityTermAE(const unsigned long freeDOFs, const double& alpha /* 0*/) const
+	{
+		throw std::runtime_error("cannot call getStressSensitivityTermAE() for this element type");
+	} // getStressSensitivityTermAE() gives error (standard) except for elements in which the function is over-written
+
+	Eigen::VectorXd element::getStressSensitivity(Eigen::MatrixXd& Lamda, const double& penal /* 1*/, const double& beta /* 1.0 / sqrt(3)*/) const
+	{
+		throw std::runtime_error("cannot call getStressSensitivity() for this element type");
+	} // getStressSensitivity() gives error (standard) except for elements in which the function is over-written
+
```

Comment:
Please state the location in the code of where this exception is thrown (for this see other exceptions)